projects/1.18/assets/tempad/tempad/lang/zh_cn.json

```
@@ -0,0 +1,31 @@
+{
+    "gui.tempad.header_line_1": "任意门 ////",
+    "gui.tempad.header_line_2": "传送程序",
+    "gui.tempad.options": "1. 选项",
+    "gui.tempad.run_program": "2. 启动传送程序",
+    "gui.tempad.wiki": "3. 官方wiki",
+    "gui.tempad.textfield": "名字:",
+    "gui.tempad.new_location": "(+) 添加新坐标",
+    "gui.tempad.add_location": "添加坐标",
+    "gui.tempad.select_location": "选择坐标:",
+    "gui.tempad.teleport": "传送",
+    "gui.tempad.delete": "删除坐标",
+    "gui.tempad.options_header": "选择一个颜色:",
+    "gui.tempad.timedoor": "任意门",
+    "tooltip.tempad.timeleft": "充满电的时间: ",
+    "tooltip.tempad.fullycharged": "完全充电好了. 冷却时间 %s ",
+    "tooltip.tempad.energy_info": "能量: %s/%s RF",
+    "tooltip.tempad.energy_cost": "每次使用消耗 %s RF ",
+    "tooltip.tempad.item_option_info": "需要使用可燃物品",
+    "tooltip.tempad.experience_level_cost": "需要 %s 级(s) 才能使用任意门",
+    "tooltip.tempad.experience_points_cost": "需要 %s 经验点才能使用任意门",
+    "item.tempad.tempad": "§6任意门",
+    "item.tempad.he_who_remains_tempad": "§6任意门",
+    "item.tempad.he_who_remains_tempad.tooltip": "“遗属”的财产",
+    "advancements.tempad.tempad": "永远...永远...",

```

```suggestion
    "advancements.tempad.tempad": "永远……永远……",
```
出处漫威《洛基》S01E06集名，没找到中文翻译